packages/viewer/src/components/scene-tree-table-layout/scene-tree-table-layout.tsx

```diff
@@ -334,10 +358,21 @@ export class SceneTreeTableLayout {
           <slot name="divider" />
         </div>
         {this.resizingColumnIndex != null && <div class="resize-overlay" />}
+        {this.isStillLoading && (
+          <slot name="loading">
+            <div class="loading">
+              <vertex-viewer-spinner size="md" />
+            </div>
+          </slot>
+        )}
```

Comment:
This component should be responsible for the layout of rows. I think the responsibility of displaying loading spinner is more appropriate in `vertex-scene-tree`.